### ISSUE TITLE
Typescript version?

### ISSUE THREAD

**Issue by kmvan** [craftgear/snake-camel#10]
```
Could not find a declaration file for module 'snake-camel'. '/path/to/node_modules/snake-camel/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/snake-camel` if it exists or add a new declaration (.d.ts) file containing `declare module 'snake-camel';`ts(7016)
```

**Comment by craftgear** [comment]
Thanks for the heads up.

As `index.d.ts` is included in `lib` directory, I don't think it is necessary to install from `@types` repository.
https://github.com/craftgear/snake-camel/blob/master/lib/index.d.ts

If it isn't the case let me know.

**Comment by craftgear** [comment]
I close this issue due to inactivity.

If you have new problems open this again.

**Comment by kmvan** [comment]
But why ts can not detect *.d.ts?

**Comment by craftgear** [comment]
Can't tell.

`tsc` doesn't complain anything.

Which version of `tsc` do you use?

**Comment by craftgear** [comment]
Ahhh, my bad.

`main` and `types` paths are wrong in package.json.
So `index.d.ts` files was missing in the published npm package.

I just published the fixed version `1.0.3`.
Can you install and try this?

Thanks.

**Comment by kmvan** [comment]
I just found out too,
Thank you.

**Comment by kmvan** [comment]
![image](https://user-images.githubusercontent.com/3839554/95297329-dcfe9100-08ac-11eb-8ff0-452d72483361.png)

Fixed in 1.0.3~
